app/server/services/sendgrid_email.js

```diff
@@ -54,37 +54,42 @@ function sendOne(template, email, templateData, callback) {
 /**
  * Send a verification email to a user, with a verification token to enter.
  * @param  {string|string[]}  email         [Email address(s) to send to]
+ * @param  {string}   firstName
  * @param  {[type]}     token    [description]
  * @param  {Function} callback [description]
  * @return {[type]}            [description]
  */
-emailService.sendVerificationEmail = function (email, token, callback) {
+emailService.sendVerificationEmail = function (email, firstName, token, callback) {
   var verifyUrlData = {
-    url: `${ROOT_URL}/verify/${token}`
+    url: `${ROOT_URL}/verify/${token}`,
+    firstName
```

Comment:
shouldnt this be firstName:firstname.
It needs to know what it's replacing.

Reply:
I remember Chris pointed out that ES6 allows for this shortcut:
Look for "Object Literals and Concise parameters" section in https://medium.freecodecamp.org/make-your-code-cleaner-shorter-and-easier-to-read-es6-tips-and-tricks-afd4ce25977c